CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 Magento Functional Testing Framework Changelog
 ================================================
+
+2.6.0
+-----
+
+* Traceability
+    * MFTF generated cest files are fully compatible for Codeception `dry-run`.
+* Modularity
+    * `mftf generate:tests` and `mftf run:test` commands now accept suite scoped test names in format `[<suitename:>testname]...`.
```

Comment:
Change the syntax example to be `[suitename:testname]...`

Reply:
Fixed